04/index.html

```diff
@@ -1,30 +1,75 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="ie=edge"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/reset.css"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/main.css"
+    >
+    <link
+        rel="stylesheet"
+        media="(min-width: 768px)"
+        href="./css/desktop.css"
+    >
+    <script
+        src="https://kit.fontawesome.com/1611b70a0e.js"
+        crossorigin="anonymous"
+    ></script>
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+
+    <nav class="nav__container">
+        <label
+            class="switcher"
+            for="menu-switcher"
+        ><i class="burger fa-solid fa-bars"></i></label>
+        <input
+            class="input"
+            type="checkbox"
+            id="menu-switcher"
+        >
+        <ul class="nav">
+            <li class="nav__item"><a href="#">Home</a>
             </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+            <li class="nav__item"><a href="#">Items</a>
+
+                <i class="fa-solid fa-chevron-down arrow-down"></i>
+                <ul class="nav__dropdown dropdown">
+                    <li class="dropdown__item"><a href="#">Item 1</a>
+                    </li>
+                    <li class="dropdown__item"><a href="#">Item 2</a>
+                        <i class="fa-solid fa-chevron-right arrow-right"></i>
+                        <ul class="nav__dropdown-second dropdown">
```

Comment:
W wersji mobilnej zagnieżdżone elementy nie wyglądają za dobrze - wychodzą poza ekran z prawej strony, warto to poprawić :)